client/app/src/main/java/com/example/criengine/Objects/Book.java

```diff
@@ -16,10 +16,10 @@
     private String isbn;
     private String status;
     private String borrower;
+    private boolean confirmationNeeded;
```

Comment:
should we initialize this to `false` in the constructor(s)? That way we wont get random nullpointer errors when we do front end logic.